ProjectTraning/Program.cs

```diff
@@ -10,6 +10,51 @@ class Program
     {
         static void Main(string[] args)
         {
+            //Task2();
+            Task3();
+            /*Console.WriteLine("Hello");
+            Console.WriteLine("Введите радиус");
+            string radiusString = Console.ReadLine();
+            int.TryParse(radiusString, out int radius);
+            Console.WriteLine($"The square of the round {Program.PerformCalculation(radius)}");
+        }
+
+
+        public static decimal PerformCalculation(int r)
+        {
+            const decimal Pi = 3.14m;
+
+            decimal square = Pi * r * r;
+
+            return square;*/
+        }
+
+
+        public static void Task1()
+        {
+            int uberflu;
+            int Identifire;
+            int u00fidentifier;
+            int myVar;
+            int myVariable;
+
+        }
+        public static void Task2()
+        {
+            decimal pi = 3.141592653m;
```

Comment:
Since these numbers are always the same, lets declare constants instead of variables.
Nobody restricted you to decide how to implement that :).